actors/market/src/types.rs

```diff
@@ -97,9 +98,9 @@ pub struct SectorDealData {
 }
 
 #[derive(Serialize_tuple, Deserialize_tuple, Debug, Clone, Eq, PartialEq)]
-pub struct ActivateDealsParams {
-    pub deal_ids: Vec<DealID>,
-    pub sector_expiry: ChainEpoch,
+#[serde(transparent)]
+pub struct BatchActivateDealsParams {
+    pub sectors: Vec<SectorDeals>,
```

Comment:
Sector deals introduces `sector_type: RegisteredSealProof` which is unecessary in this PR but will be used in https://github.com/filecoin-project/builtin-actors/pull/1312 to optimise `ProveReplicaUpdates`